### ISSUE TITLE
`collateral` - block adding collaterals if it will make DFI < 50%

### ISSUE THREAD

**Issue by thedoublejay** [DeFiCh/wallet#1518]

**Comment by kyleleow** [comment]
If we are blocking user, i think we should inform user upfront in maybe edit collateral screen, with a short message that says something along the line like 
> "Vault's collateral has to consists at least 50% of DFI to borrow loan token"

So that we can save user a few steps trying to add collaterals that will be blocked.

Regarding the exact blocking implementation, i believe it should be done in this screen? We will block the `Add token ..` button and display a validation text?
cc: @jtlntno-cake @thedoublejay 
![image](https://user-images.githubusercontent.com/40191153/144537385-c5085d36-60a0-4dfe-90c7-234806cebd33.png)

**Comment by thedoublejay** [comment]
Yup the change would be on this page @kyleleow not on the take loan screen
